src/patterns/error-status-pages/examples/timeout-error/index.njk

```diff
@@ -17,13 +17,24 @@ layout: none
 } %}
 
 {% block main %}
-    <h1>Your session has timed out due to inactivity</h1>
 
-    {% call onsPanel({
-        "classes": "u-mb-m"
-    }) %}
-        <p>To protect your information we have timed you out.</p>
-    {% endcall %}
+    <h1>Sorry, you need to start again</h1>
```

Comment:
I've just realised this is one of two variations @LauraC119 .  This is the "transaction" timeout page, but there's also the "signed into a survey" timeout page: https://census-website-respond.netlify.app/end-to-end/session-timeout-rh-authentication-v2/ 
So we'll need to either show both examples, or document what you should put in the other case.  Considering the survey scenario is the most common, we should probably use that example.
We might then want to document what a service should use if the user isn't signed into the service via an access code e.g. "Sorry, you'll need to start again" (Transaction e.g. RH) or "Sorry, you'll need to sign in again" (RAS service)